rest-api/offline/metrics_export.py

```diff
@@ -27,11 +27,14 @@
 _PARTICIPANT_SQL_TEMPLATE = """
 SELECT p.participant_id, ps.date_of_birth date_of_birth,
   (SELECT ISODATE[MIN(bo.created)] FROM biobank_order bo
-    WHERE bo.participant_id = p.participant_id) first_order_date,
+    WHERE bo.participant_id = p.participant_id
+    AND bo.order_status != 2) first_order_date,
```

Comment:
I think you need (bo.order_status is null or bo.order_status != 2)... or if you prefer, bo.order_status <> 2.